src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -210,8 +210,8 @@ def _merge_new_units(self, target_path, corpus_dir, new_corpus_dir,
       new_units_added = new_corpus_len - old_corpus_len
 
       stat_overrides.update(result.stats)
-    except MergeError:
-      logs.log_warn('Merge failed', target=os.path.basename(target_path))
+    except (MergeError, engine.TimeoutError):
+      logs.log_warn('Merge failed.')
```

Comment:
This target path is unneeded since fuzz target is in log variable.

Reply:
Thanks!

Reply:
Does this mean we lose the output from https://github.com/google/clusterfuzz/blob/7d5cec7eed73366bcbcf733df6af02f362545f8a/src/python/bot/fuzzers/libFuzzer/engine.py#L484, or is it logged anyways somehow?

Reply:
Probably you can add it as part of https://github.com/google/clusterfuzz/pull/1289#pullrequestreview-333336737

Reply:
Done!